scripts/update-api.js

```diff
@@ -0,0 +1,3 @@
+require('./tasks/api-extractor')({
+  args: '--local'
```

Comment:
Why do you need this file?

Reply:
This makes it easy to add the update-api task to the utilities, merge-styles, styling, and oufr packages (as well as any packages that we decide to enable api-extractor for in future).

Reply:
@kkjeer FYI we normally update the API automatically as part of a local build.  Having people run a special command to update the API seems like an extra step that they can forget to do.

---
In reply to: [226745780](https://github.com/OfficeDev/office-ui-fabric-react/pull/6760#discussion_r226745780) [](ancestors = 226745780)

Reply:
To summarize our offline discussion: we can try this approach out and see what feedback people have, and revisit this issue in the future.